docs/guide/docs/installation.md

```diff
@@ -30,7 +30,7 @@ Alternatively, you can also start it from the terminal:
 > ./bp
 ```
 
-The first time you run Botpress, the built-in modules take some time to install. This will only happen the first time you run Botpress, subsequent runs will be much faster.
+The first time you run Botpress, the built-in modules take some time to install. However, subsequent runs will be much faster.
```

Comment:
Botpress is also way slower on windows than on any other plateform (because of Windows Defender). There is an easy way to solve this. Should we add it in the documentation?

Reply:
There is also some issues running it on Mac the first time. You need to approve alot of stuff in your system preferences (security). Always bugs me when I have to do it.